demos/background_subtraction_demo/python/background_subtraction_demo.py

```diff
@@ -0,0 +1,296 @@
+    }
+
+    model = get_instance_segmentation_model(model_adapter, configuration)
+
+    person_id = -1
+    for i, label in enumerate(labels):
+        if label == 'person':
+            person_id = i
+            break
+    assert person_id >= 0, 'Person class did not find in labels list.'
+
+    model.log_layers_info()
+
+    pipeline = AsyncPipeline(model)
+
+    next_frame_id = 0
+    next_frame_id_to_show = 0
+
+    metrics = PerformanceMetrics()
+    render_metrics = PerformanceMetrics()
+    presenter = None
+    output_transform = None
+    video_writer = cv2.VideoWriter()
+    while True:
+        if pipeline.is_ready():
+            # Get new image/frame
+            start_time = perf_counter()
+            frame = cap.read()
+            bgr = target_bgr.read() if target_bgr is not None else None
+            if frame is None:
+                if next_frame_id == 0:
+                    raise ValueError("Can't read an image from the input")
+                break
+            if next_frame_id == 0:
+                output_transform = OutputTransform(frame.shape[:2], args.output_resolution)
+                if args.output_resolution:
+                    output_resolution = output_transform.new_resolution
+                else:
+                    output_resolution = (frame.shape[1], frame.shape[0])
+                presenter = monitors.Presenter(args.utilization_monitors, 55,
+                                               (round(output_resolution[0] / 4), round(output_resolution[1] / 8)))
+                if args.output and not video_writer.open(args.output, cv2.VideoWriter_fourcc(*'MJPG'),
+                                                         cap.fps(), tuple(output_resolution)):
+                    raise RuntimeError("Can't open video writer")
+            # Submit for inference
+            pipeline.submit_data(frame, next_frame_id, {'frame': frame, 'start_time': start_time})
+            next_frame_id += 1
+        else:
+            # Wait for empty request
+            pipeline.await_any()
+
+        if pipeline.callback_exceptions:
+            raise pipeline.callback_exceptions[0]
+        # Process all completed requests
+        results = pipeline.get_result(next_frame_id_to_show)
+        if results:
+            objects, frame_meta = results
+            if args.raw_output_message:
+                print_raw_results(objects, next_frame_id_to_show)
+            frame = frame_meta['frame']
+            start_time = frame_meta['start_time']
+            rendering_start_time = perf_counter()
+            frame = render_results(frame, objects, output_resolution, bgr, person_id,
+                                   args.blur_bgr, args.show_with_original_frame)
+            render_metrics.update(rendering_start_time)
+            presenter.drawGraphs(frame)
+            metrics.update(start_time, frame)
+
+            if video_writer.isOpened() and (args.output_limit <= 0 or next_frame_id_to_show <= args.output_limit-1):
```

Comment:
Why don't just `next_frame_id_to_show < args.output_limit`

Reply:
Just made like in [object_detection_demo.py](https://github.com/openvinotoolkit/open_model_zoo/blob/master/demos/object_detection_demo/python/object_detection_demo.py#L257).

Reply:
Got it, let it be